managed-ledger/src/main/java/org/apache/bookkeeper/mledger/util/StatsBuckets.java

```diff
@@ -73,6 +73,19 @@ public void refresh() {
         this.count = count;
     }
 
+    public void collect() {
+        long count = 0;
+        sum = sumCounter.sum();
+
+        for (int i = 0; i < buckets.length; i++) {
+            long value = buckets[i].sum();
+            count += value;
+            values[i] = value;
+        }
+
+        this.count = count;
```

Comment:
There seems to be a problem. For Histogram, `count` is the times of add/observe method invoked